apps/desktop/src/renderer/components/Welcome.module.css

```
@@ -0,0 +1,115 @@
+/* Welcome — First-run onboarding overlay */
+
+@keyframes fadeIn {
```

Stylelint (17.7.0)

[error] Expected keyframe name "fadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)